### PR TITLE
Reaper

### PR DESCRIPTION
**Pull request by bbalser**

### CODE REVIEW

File: apps/reaper/mix.lock [hunk 1]
```diff
@@ -0,0 +1,96 @@
+%{
```
Comment (ManApart) [inline apps/reaper/mix.lock]: I believe you can drop this file

Comment (ManApart) [reply apps/reaper/mix.lock]: This is jessie btw lol

**Comment by mcsearchin** [comment]
It looks like this is just moving the files over with no other changes sans configuration. Is that correct?

**Comment by jdenen** [comment]
That's what I see @mcsearchin 

That's what he did for `valkyrie` too.

**Comment by ManApart** [comment]
Even when I don't review a PR, I review that PR. lol

**Pull request merged**